src/pocketmine/scheduler/AsyncTask.php

```diff
@@ -216,14 +201,44 @@ public function onProgressUpdate(Server $server, $progress){
 	}
 
 	/**
-	 * Call this method from {@link AsyncTask#onCompletion} to fetch the data stored in the constructor, if any, and
-	 * clears it from the storage.
+	 * Saves mixed data in ServerScheduler's object storage on the main thread. You may use this to retain references to
+	 * objects or arrays which you need to access in {@link AsyncTask#onCompletion} which cannot be stored as a
```

Comment:
onCompletion() or onProgressUpdate().

Reply:
I think it's best not to introduce more confusion over fetch vs peek. This documents behaviour and the most common use, it's up to developers to decide how to use it.